### PR TITLE
added Rebus project

### PR DESCRIPTION
**Pull request by mookid8000**

### CODE REVIEW

File: _data/projects/rebus.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- "Azure Service Bus"
+- "Azure Storage Queues"
+- "distributed"
+- "messaging"
+- "service bus"
+- "soa"
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/rebus-org/Rebus/labels/up-for-grabs
+stackoverflow:
```
Comment (shiftkey) [inline _data/projects/rebus.yml]: cc @dahlbyk @jrusbatch for feels on extra topics (I like the SO option)

Comment (jrusbatch) [inline _data/projects/rebus.yml]: I like SO, too. I'm not sure about `blog` though, that seems like it could overlap with `site`.

Comment (shiftkey) [inline _data/projects/rebus.yml]: That seems fair. What about, as a first step, we make `stackoverflow` an optional element (I can spec it out as part of this review) and then work out how to surface it through the UI?

File: _data/projects/rebus.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  name: Rebus questions on StackOverflow
```
Comment (shiftkey) [inline _data/projects/rebus.yml]: @mookid8000 I don't think we need this element (unless we do something more fancy than link to SO). Thoughts?

File: _data/projects/rebus.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  link: http://stackoverflow.com/questions/tagged/rebus
+blog:
```
Comment (shiftkey) [inline _data/projects/rebus.yml]: As mentioned above, I don't know if this adds much value on top of the `site` link above.

Comment (dahlbyk) [inline _data/projects/rebus.yml]: Agreed

File: _data/projects/rebus.yml (hunk 2 above)
Comment (dahlbyk) [inline _data/projects/rebus.yml]: I don't think we need it

Comment (jrusbatch) [inline _data/projects/rebus.yml]: Agreed.

**Comment by mookid8000** [comment]
I have removed the extra links now :smiley:

**Comment by dahlbyk** [comment]
:sparkles:

**Comment by mookid8000** [comment]
:zap: :clap: :tada:

**Pull request merged**